codingTest_kjy/수박수박수박수박수.js

```diff
@@ -0,0 +1,10 @@
+function solution(n) {
+    let answer = '';
+    for(let i=1; i<=n; i++)
+        if(i% 2=== 0){
+        answer +="박"
+}else{
+        answer +="수"
+}
+return answer
+}
```

Comment:
```suggestion
function solution(n) {
  let answer = '';
  for (let i = 1; i <= n; i++) { //중괄호!!!!!!!!!!!!!
    if (i % 2 === 0) {
      answer += '박';
    } else {
      answer += '수';
    }
  }
  return answer;
}
```
코드 컨벤션 꼭 지켜주시고 위에 반복문, 조건문 할 때 중괄호 꼭 기억해주세요 중괄호중괄호중괄호중괄호중괄호중괄호중괄호중괄호중괄호중괄호중괄호중괄호중괄호중괄호중괄호중괄호

Reply:
중괄호 기억하겠습니다!